spec/system/projects_spec.rb

```diff
@@ -8,34 +8,34 @@
     visit root_path
   end
 
-  def create_project
+  def create_project(options)
```

Comment:
```suggestion
  def create_project(name:, description:)
```

のようにキーワード引数にした方がtypoに気付きやすくなるのでベターです。